From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sat, 12 Sep 2020 02:23:37 +0200
Subject: [PATCH] tests: svthevcenc: Fix test_encode_simple

Pick the same I420 format the other test use. Without this, the source
picks AYUV64, which fails.
---
 tests/check/elements/svthevcenc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/check/elements/svthevcenc.c b/tests/check/elements/svthevcenc.c
index d1234fc27..1b938e10c 100644
--- a/tests/check/elements/svthevcenc.c
+++ b/tests/check/elements/svthevcenc.c
@@ -29,7 +29,7 @@ GST_START_TEST (test_encode_simple)
   h = gst_harness_new_parse ("svthevcenc speed=9 bitrate=1000 ! h265parse");
 
   gst_harness_add_src_parse (h, "videotestsrc is-live=true ! "
-      "capsfilter caps=\"video/x-raw,width=320,height=240,framerate=25/1\"",
+      "capsfilter caps=\"video/x-raw,format=I420,width=320,height=240,framerate=25/1\"",
       TRUE);
 
   /* Push 25 buffers into the encoder */
